### PR TITLE
feat: improve job offer list

### PR DESCRIPTION
**Pull request by miedziak**
- add headers
- prepare to separate page convention

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
     "react-share": "4.1.0",
     "react-spring": "8.0.27",
     "resize-observer": "1.0.0",
+    "slugify": "^1.4.0",
     "styled-system": "5.1.5",
```
Comment (LWronka) [inline package.json:76]: Bind to version

File: scripts/get-markdown-pages.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          }
+        }
+      }
+    }
+  `;
+
+const getMarkdownPages = async (graphql) => {
+  const mdxResults = await graphql(allMdxQuery);
+  if (mdxResults.errors) {
+    console.log(mdxResults.errors); // eslint-disable-line no-console
```
Comment (LWronka) [inline scripts/get-markdown-pages.js:26]: If it is error use`console.error`

File: src/assets/slack-logo.svg [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 64 (101010) - https://sketch.com -->
+    <title>214D8813-3F37-4E86-803F-76FBC3A265FD</title>
+    <desc>Created with sketchtool.</desc>
```
Comment (LWronka) [inline src/assets/slack-logo.svg:5]: That content should be removed

File: src/components/pages/PageSection.js [hunk 4]
```diff
@@ -2,24 +2,22 @@ import React from 'react';
 import styled from '@emotion/styled';
 import { space, color, layout } from 'styled-system';
 import PropTypes from 'prop-types';
+import {darken} from "polished";
```
Comment (LWronka) [inline src/components/pages/PageSection.js:5]: One time you have bracket space in other place you have bracket without space. Fix eslint formatting.

File: src/components/pages/team/JobList.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+`;
+
+const JobList = ({ title, jobs }) => (
+  <Container>
+    <AnimateOnScroll slideIn="down">
+      <Category>{title}</Category>
+    </AnimateOnScroll>
+    {jobs.map((job, index) => (
+      <AnimateOnScroll
+        key={job.createdAt.toString() + job.id}
```
Comment (LWronka) [inline src/components/pages/team/JobList.js:29]: `job.id` isn't enough ?

File: src/components/pages/team/JobList.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        slideIn="down"
+        delay={100 * index}
```
Comment (LWronka) [inline src/components/pages/team/JobList.js:31]: Here is needed establish max value, let's make util and make there linear table with values - function could take - `index, itemsLength, maxValue`

File: src/components/pages/team/JobPage.js [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+            slideIn="down"
+            delay={100 * index}
+          >
+            <JobItem job={categoryJob} />
+          </AnimateOnScroll>
+        ))}
+      </PageSection>
+    </>
+  );
+};
```
Comment (LWronka) [inline src/components/pages/team/JobPage.js:198]: Fo better readability I think that component could be divide into few smaller components like:
- `Header` with back button
- `Job List`
- `Not Suitable offer`

File: src/components/pages/team/JoinTeam.js [hunk 8]
```diff
@@ -1,22 +1,37 @@
...
 const AnchorPointer = styled('div')`
   padding-bottom: 8rem;
 `;
 
-const JoinTeam = ({ jobs, updatedJobs }) => {
-  const items = updatedJobs.length > 0 ? updatedJobs : jobs;
+const separateJobsByCategories = jobs => {
+  return jobs.reduce((acc, job) => {
+    const categoryIndex = acc.findIndex(
+      item => item.name && item.name === job.categories.team
```
Comment (LWronka) [inline src/components/pages/team/JoinTeam.js:18]: duplicate item.name

**Comment by miedziak** [comment]
merged

**Pull request closed without merging**